packages/components/button/src/button.vue

```diff
@@ -50,6 +58,7 @@ export default defineComponent({
   emits: buttonEmits,
 
   setup(props, { emit, slots }) {
+    const buttonRef = ref(null)
```

Comment:
We can leave the param empty like 

`const buttonRef = ref<HTMLButtonElement>()`

The type of this variable would be 

```typescript
Expect<buttonRef>(Ref<HTMLButtonElement | undefined>)
```